src/en/components/radio/code.md

```diff
@@ -20,14 +20,4 @@ The radio helps users to make a choice by limiting their options.
 - Use the fieldset `legend` and `hint` attributes for the group name and instructions.
 - For required groups, set the `required` attribute in the fieldset. Setting the `required` attribute applies validation and error handling to the radio group.
 
-{% include "partials/getcode.njk" %}
-
-<iframe
-  title="Overview of gcds-radio properties and events."
-  src="https://cds-snc.github.io/gcds-components/iframe.html?viewMode=docs&demo=true&singleStory=true&id=components-radio--events-properties&lang=en"
-  width="1200"
-  height="1770"
-  style="display: block; margin: 0 auto;"
-  frameBorder="0"
-  allow="clipboard-write"
-></iframe>
+{% include "partials/getcode.njk" %}
```

Comment:
The getcode partials can also be removed since it provides supporting documentation for the storybook frome